script/chromium-rebase-l10n.py

```diff
@@ -8,16 +8,108 @@
 import argparse
 import os.path
 import sys
-
-from lxml import etree  # pylint: disable=import-error
+import glob
 from lib.l10n.grd_utils import (get_override_file_path,
                                 textify,
                                 update_braveified_grd_tree_override,
                                 write_xml_file_from_tree,
                                 write_braveified_grd_override)
+from lxml import etree  # pylint: disable=import-error
+
+BRAVE_SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+SRC_SOURCE_ROOT = os.path.abspath(os.path.dirname(BRAVE_SOURCE_ROOT))
+
+sys.path.insert(1, os.path.join(SRC_SOURCE_ROOT, 'tools/grit'))
+
+from grit.extern import tclib  # pylint: disable=import-error,wrong-import-position
+
+# Map of google_chrome_strings.grd resources ids to migrate to brave_strings.grd
+# The resources and all translations will be migrated to grd and xtb files.
+# key - id in google_chrome_strings.
+# value - new id in brave_stirngs.
+GOOGLE_CHROME_STRINGS_MIGRATION_MAP = {
+    'IDS_SHORTCUT_NAME_BETA': 'IDS_BRAVE_SHORTCUT_NAME_BETA',
+    'IDS_SHORTCUT_NAME_DEV': 'IDS_BRAVE_SHORTCUT_NAME_DEV'
+}
+
+
+def write_etree_content(xml_tree, source_string_path):
+    transformed_content = etree.tostring(xml_tree,
+                                         pretty_print=True,
+                                         xml_declaration=True,
+                                         encoding='UTF-8')
```

Comment:
I think everywhere else we use 
<pre>
xtb_content = (b'<?xml version="1.0" ?>\n' +
            lxml.etree.tostring(xtb_tree, pretty_print=True,
                xml_declaration=False, encoding='utf-8')
</pre>
We should probably do the same here so the xml declaration doesn't get changed all the time.

Reply:
This breaks xtb because inserts empty line at top also removes <xml> declaration on the first line, it doesnt look correct

Reply:
Sorry, the Github formatting hid the actual code:

<pre>
xtb_content = (b'&lt;?xml version="1.0" ?&gt;\n' +
            lxml.etree.tostring(xtb_tree, pretty_print=True,
                xml_declaration=False, encoding='utf-8')
</pre>

Reply:
it works, done, thanks